tests/integration.rs

```diff
@@ -173,23 +191,63 @@ fn stamp_file() -> TestResult {
 
     // For some reason, we delete the stamp file after `--file` :(
     // Recreate it.
-    run(sweep(&["--stamp"]));
+    run(sweep(&["--stamp"]), target.path());
 
     let actual_cleaned = count_cleaned(&target, args, size)?;
     assert_eq!(actual_cleaned, expected_cleaned);
 
     Ok(())
 }
 
+#[test]
+fn empty_project_output() -> TestResult {
+    let (_size, target) = build("sample-project")?;
+
+    let assert = run(
+        sweep(&["--maxsize", "0"]).current_dir(test_dir().join("sample-project")),
+        target.path(),
+    );
+
+    let output = std::str::from_utf8(&assert.get_output().stdout).unwrap();
+
+    let regex_skip = r#".+?"#;
+
+    let pattern = unindent(&format!(
+        r#"\[DEBUG\] cleaning: "/tmp/{regex_skip}" with remove_older_until_fits
+        \[DEBUG\] size_to_remove: {regex_skip}
+        \[DEBUG\] Sizing: "/tmp/{regex_skip}/debug" with total_disk_space_in_a_profile
```

Comment:
ah, and like CI shows this needs to be `".*/debug"` instead of assuming it's in /tmp.